test/test_table.cpp

```diff
@@ -1002,7 +990,6 @@ void setup_multi_table(Table& table, size_t rows, std::vector<ObjKey>& keys, std
     auto string_col = table.add_column(type_String, "string");               //  4
     auto string_long_col = table.add_column(type_String, "string_long");     //  5
     auto string_big_col = table.add_column(type_String, "string_big_blobs"); //  6
-    auto string_enum_col = table.add_column(type_String, "string_enum");     //  7 - becomes StringEnumColumn
```

Comment:
Shouldn't the numbers in the comment being decreased as well?

Reply:
I think the numbers dates back to the days where we referenced columns by their index and not by the stable ColKey. But in a sense it is ok to leave the numbers as the ColKeys will not change.